slatedb/src/db/builder.rs

```diff
@@ -611,6 +613,11 @@ impl<P: Into<Path>> DbBuilder<P> {
                 .await?;
         }
 
+        // Preload meta cache (index and filter) if configured
+        if let Err(e) = inner.preload_meta_cache().await {
```

Comment:
`preload_metadata_cache()` etc etc elsewhere. Trying to keep things very clear. "meta_cache" isn't very obvious to users.